### PR TITLE
Create 2019-02-01-Episode-13-Jupyter_Ecosystem.md

### PR DESCRIPTION
**Pull request by bsodenkamp**

### CODE REVIEW

File: _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Introductions
+
+Welcome to Open Source Directions, the webinar that informs you about the future of open source projects.
+This episode, the project being highlighted is [The Jupyter Ecosystem](https://jupyter.org/)
+
+* **Host:** Anthony Scopatz
+* **Co-Host:** David Charboneau
+* **Guest 1:** Carol Willing, is a Jupyter Steering Council member, core developer with a focus on Jupyter's use in learning (over 5 years), and a Python core developer.
+* **Guest 2:** Matthias Bussonnier,  by Day he’s a Research facilitator at the University of California, Merced, and by Night a Pythonista (mostly), and one of the early founders of Jupyter and a core developer of Jupyter as well as IPython.
```
Comment (scopatz) [inline _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md]: Should be "by day he is" and "by night he is a"

File: _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
```
Comment (scopatz) [inline _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md]: Should be "IDE-like" and "RStudio"

File: _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+
+### Demo
+
+[Demo at timestamp 34:13](https://www.youtube.com/watch?v=HzyfFSLfHiQ)
```
Comment (scopatz) [inline _posts/2019-02-01-Episode-13-Jupyter_Ecosystem.md]: This link should be to the timestamped location, ie https://youtu.be/HzyfFSLfHiQ?t=2053

**Comment by scopatz** [comment]
Hey @bsodenkamp - thanks! Please let me know here in the comments when it is ready to take another look.

**Comment by bsodenkamp** [comment]
Yes, I made those edits and it should be ready for review.  Sorry about that.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**